test/e2e-v2/cases/php/expected/trace-info-detail.yml

```diff
@@ -15,30 +15,34 @@
 
 spans:
   {{- contains .spans }}
-  - traceid: {{ .traceid }}
-    segmentid: {{ .segmentid }}
+  - traceid: "{{ .traceid }}"
+    segmentid: "{{ .segmentid }}"
     spanid: {{ .spanid }}
     parentspanid: {{ .parentspanid }}
     refs: []
     servicecode: php
     serviceinstancename: {{ notEmpty .serviceinstancename }}
     starttime: {{ gt .starttime 0 }}
     endtime: {{ gt .endtime 0 }}
-    endpointname: /php/info
+    endpointname: POST:/php/info
     type: Entry
-    peer: {{ notEmpty .peer }}
+    peer: ""
     component: PHP
     iserror: false
     layer: Http
     tags:
       {{- contains .tags }}
       - key: url
         value: {{ notEmpty .value }}
+      - key: http.method
+        value: {{ notEmpty .value }}
+      - key: http.status_code
+        value: {{ notEmpty .value }}
```

Comment:
Maybe here can be a specific value?

Reply:
Hi sir, after I locked the value and ran the local test, the e2e testing was not going to pass because the order of the testing was wrong. I think it is a bug from the e2e testing tool. Can I change it back to {{notEmpty .value}} instead of a fixed value?

Reply:
Here is the screenshot: 
![WechatIMG1366](https://github.com/apache/skywalking/assets/101088629/162a82b6-7ea5-4a1a-87cd-2c8adee0b63a)

Reply:
@kezhenxu94 Any suggestion here?

Reply:
@simonluo345 consider pushing your changes so that I can see where is the potential  problem, one thing I noticed from the screenshot is that you might wrote some of the expected rules twice like this 

<img width="1276" alt="image" src="https://github.com/apache/skywalking/assets/15965696/4227eade-d8bf-4662-ac61-9d572910147e">

Not sure though, would be helpful to push the changes here

Reply:
If I change it back to {{notEmpty .value}} everything is ok. 
![WeChat79e1f3369a5bd993a77c6e7366d2dbb2](https://github.com/apache/skywalking/assets/101088629/4b9d7169-acd4-4495-a0c7-af664d7ace65)

Reply:
@simonluo345 Your reply doesn't make sense. We are asking whether your expected data is correct.
Making tests passed is not the purpose. Write the tests correctly and really verify codes, these are the purpose.

Reply:
@wu-sheng Thank you for pointing out this. I misunderstood the question.

Reply:
@kezhenxu94 just pushed the changes, could you please take a look?